decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
While I agree with leaving a comment here explaining what's going on (this is a non-common use of a Rails feature, I guess, I'm aware of that), I'd change the code to use `to_str`. What we used to have is really ugly (`"#{content_tag(...)}"`), `to_str` looks cleaner to me.

What do you think @deivid-rodriguez? :)